apiary.apib

```diff
@@ -1222,6 +1224,160 @@ This endpoint gives data for a blockchain recorded credential. It will help you
           }
 
 
+## BlockChain Credential V2 [/v2/credentials/{id}/blockchain_data]
+
+###Verify BlockChain Credential [GET]
+
+This endpoint gives data for a blockchain recorded credential.
+It will help you verify authenticity of the credential on blockchain.
+This endpoint would return different responses based on the Blockchain (*Bitcoin* or *Ethereum*) that it's issued in.
+
+[Verify Blockchain record of a Credential - Video Walkthrough](https://www.loom.com/share/23621ac848c44a77ac1eda0eeb975d93?sid=c10228b9-6317-477b-b903-20ed78ce6b76)
+
++ Parameters
+    + id(string) - Encoded id or UUID of the credentials. unique id in the credentials url.
+
+
++ Request (application/json)
+
+    + Attributes
+        + key (string, optional) - If private credentials provide a key
+
++ Response 200 (application/json)
```

Comment:
One confusion, I see we have 2 responses, both of these responses are visible on https://accrediblecredentialapi.docs.apiary.io/# ?

Reply:
Currently we are showing one response (for bitcoin) in the docs.
For the new API, we are showing both the responses (bitcoin & ethereum)